code/modules/holoparasite/holoparasite_builder.dm

```diff
@@ -380,6 +380,7 @@
 	var/mob/dead/observer/candidate = pick(candidates)
 	var/mob/living/simple_animal/hostile/holoparasite/holoparasite = new(user, candidate.key, holopara_name, theme, accent_color, notes, user.mind, saved_stats)
 	var/datum/antagonist/holoparasite/holopara_antag = holoparasite.mind.add_antag_datum(new /datum/antagonist/holoparasite(user.mind.holoparasite_holder(), saved_stats, theme))
+	saved_stats = new
```

Comment:
why does this work, what

Reply:
Otherwise, `holopara.stats` and `builder.stats` would reference the _same stats instance_ - so after we create a holopara with the stats we want, we set `builder.stats` to a new instance.

Reply:
Does the builder need a new stats object? Can't you just set it to null?

Reply:
I mean, sure, but this is the easier fix as it never expects stats to be null, as stats is usually initialized during unit, and it's possible to varedit these to be reusable

Reply:
Alright then